hydro/CPL/WRF_cpl/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+add_library(hydro_wrf_cpl STATIC
+	wrf_drv_HYDRO.F90
+        module_wrf_HYDRO.F90)
+
+add_dependencies(hydro_wrf_cpl
+  hydro_mpp
+  hydro_utils
+  hydro_debug_utils
+  hydro_data_rec
+  hydro_driver
+  hydro_orchestrator
+  ${PROJECT_NAME}_Core)
+
+target_include_directories(hydro_wrf_cpl
```

Comment:
This is missing MPI includes / "linking"
Not sure if you want to get that from link against `WRF_Core` or manually specify the specific things you need as currently done

Reply:
Added MPI linking

Reply:
Still getting issues on this part. I don't think `add_dependencies()` adds the necessary includes

Reply:
@scrasmussen Pinging just in case you haven't seen this re-opened